app/src/main/java/cm/aptoide/pt/analytics/view/AnalyticsActivity.java

```diff
@@ -26,9 +25,7 @@ public boolean is_resumed() {
 
   @Override protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
-    getActivityComponent().inject(this);
-    Analytics.Lifecycle.Activity.onCreate(this);
-
+    analyticsManager = ((AptoideApplication) getApplicationContext()).getAnalyticsManager();
```

Comment:
Can we inject the AnalyticsManager dependency?

Reply:
Fixed. The problem was that it was assuming the class was ActivityResultNavigator with "this", as a result it wasn't injecting the dependecies because the ActivityResultNavigator was already injected.